contracts/core/Allo.sol

```diff
@@ -594,6 +602,18 @@ contract Allo is
         emit BaseFeeUpdated(baseFee);
     }
 
+    /// @notice Adds a pool manager
+    /// @dev Internal function used to add a pool manager.
+    /// @param _poolId The ID of the pool
+    /// @param _manager The address to add
+    function _addPoolManager(uint256 _poolId, address _manager) internal {
```

Comment:
Now that we have this function maybe we should use `_addPoolManager` for https://github.com/allo-protocol/allo-v2/blob/87f18bbad4302f1c28d7ad669bddad301c499503/contracts/core/Allo.sol#L458-L461

Reply:
true! nice notice

Reply:
nice catch!